### PR TITLE
Trick Codecov to ignore site-packages

### PR DESCRIPTION
**Pull request by kalefranz**
@stevepeak let's use this PR.  Pointed at the 4.3.x branch with is the current release branch, and also passing tests.  I'll post links to Travis Appveyor, and Codecov runs for each commit.

### CODE REVIEW

**Comment by kalefranz** [comment]
* https://travis-ci.org/conda/conda/builds/186235370
* https://ci.appveyor.com/project/ContinuumAnalyticsFOSS/conda/build/1.0.2168
* https://codecov.io/gh/conda/conda/commit/3d5818baf492d0b026cd015e6744cd19e72aa425

File: .codecov.yml [hunk 1]
```diff
@@ -10,3 +10,6 @@ ignore:
   - conda/gateways/adapters/ftp.py
   - conda/gateways/adapters/s3.py
   - tests/.*
+
+fixes:
+  - ".*/Lib/site-packages/.*::"
```
Comment (kalefranz) [inline .codecov.yml]: @stevepeak should we also add

```
  - ".*/lib/python\d\.\d/site-packages/.*::"
```

?

**Comment by kalefranz** [comment]
Also supersedes https://github.com/conda/conda/pull/4122

File: .codecov.yml (hunk 1 above)
Comment (stevepeak) [inline .codecov.yml]: Darn.. this trick did not work. I'll need to come up with something else.

**Pull request closed without merging**